Source/WebCore/page/Quirks.cpp

```diff
@@ -1041,6 +1041,17 @@ bool Quirks::shouldAvoidPastingImagesAsWebContent() const
 #endif
 }
 
+bool Quirks::shouldInjectYouTubeFullsceenStyles() const
+{
+    if (!needsQuirks())
+        return false;
+
+    if (!m_shouldInjectYouTubeFullsceenStyles)
+        m_shouldInjectYouTubeFullsceenStyles = m_document->url().host() == "youtube.com"_s || m_document->url().host().endsWith(".youtube.com"_s);
```

Comment:
There is a function now to target domain name. Use it when possible.
https://github.com/WebKit/WebKit/blob/a501e76d923208b49d1f5bb2757035bce36455b7/Source/WebCore/page/Quirks.cpp#L111-L114

Reply:
I want this quirk to also apply to embedded YouTube videos, so I believe Quirks::isDomain wouldn't work here since it only compares against the top document?

Reply:
hmm indeed. You might have uncovered a mistake in my previous PR. I will double check. Go with the code you proposed, and I'll change it later if necessary or make it more flexible. Thanks!